packages/dev/core/src/Engines/engineCapabilities.ts

```diff
@@ -121,4 +121,6 @@ export interface EngineCapabilities {
     supportFloatTexturesResolve: boolean;
     /** Defines if RG11B10UFloat texture format is color renderable */
     rg11b10ufColorRenderable: boolean;
+    /** Defines if EXT_texture_norm16 is available */

```

Comment:
I think it's better to list the new supported formats when the boolean is true than giving the name of the extension, as the extension is WebGL specific.

Reply:
Ok, I added the new texture formats to the extension name, so if someone searches for it, she would find it in either case.